code/controllers/subsystems/zcopy.dm

```diff
@@ -115,7 +115,7 @@ SUBSYSTEM_DEF(zcopy)
 	zlev_maximums = new(world.maxz)
 	var/start_zlev = 1
 	for (var/z in 1 to world.maxz)
-		if (!HasAbove(z))
+		if (!SSmapping.multiz_levels[z][Z_LEVEL_UP])
```

Comment:
Might be worth to add a HAS_Z_ABOVE / BELOW Macro that's consistently used everywhere?